WalletWasabi/WabiSabi/ProtocolContants.cs

```diff
@@ -5,5 +5,24 @@ public static class ProtocolConstants
 		public const int CredentialNumber = 2;
 		public const ulong MaxAmountPerAlice = 4_300_000_000_000ul;
 		public const uint MaxVsizePerAlice = 255;
+
+		public const string WabiSabiProtocolIdentifier = "WabiSabi_v1.0";
+		public const string DomainStrobeSeparator = "domain-separator";
+
+		// Round hashing labels
+		public const string RoundStrobeDomain = "round-parameters";
```

Comment:
nameof cannot work here? This is a rename maintenance burden.

Reply:
No, `nameof` is not good here, that's what I did the first but it is dangerous because if someone renames a property Wasabi will stop working and it will be extremely hard to figure out why. These strings were created precisely to separate the strobe labels from the property names.

Reply:
ACK